blueprint/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Using Scripts to screenpop a URL when after inactivity when a conversation ends on Genesys Cloud
+author: zino.onokpise
+indextype: blueprint
+icon: blueprint
+category: 6
+summary: |
+  This Genesys Cloud Developer Blueprint explains how to create an embeddable app in scripts to set up a notification channel and listen for events. In this blueprint, a subscription is made to a user's active conversations. Events from this topic are then used to track the state of the conversation and alerts to the user when a conversation has ended and the chat should be closed
+---
+
+This Genesys Cloud Developer Blueprint explains how to create an embeddable app in scripts, set up a notification channel and listen for events. In this blueprint, a subscription is made to a user's active conversations. Events from this topic are then used to track the state of the conversation and alerts to the user when a conversation has ended and the chat should be closed
+
+### Scenario
+
+You notice that your agents are not always on call but calls are getting stacked on queues. You find out its because agents are not closing their chats when a conversation ends. So you need a mechanism to automatically prompt your agents to wrap up the chat. ([Inspired by URL Open at conversation end](https://developer.genesys.cloud/forum/t/url-open-at-conversation-end/15204))
+
+### Solution
+
+- Create a script that will open an IFrame and run an embedded client side app
+- Set up Implicit Grant Login
+- Connect to the Genesys Notification Service and subscribe to the agent's active conversations
+- Screen pop a URL after a timeout has been reached starting from when a conversation ends
+
+### Solution components
+
+- **Genesys Cloud CX** - A suite of Genesys Cloud services for enterprise-grade communications, collaboration, and contact center management. In this solution, you use an Architect inbound message flow, a Genesys Cloud integration, a Genesys Cloud queue, web messaging configuration, and web messaging deployment.
+- **Web messaging and Messenger** - The Genesys Cloud messaging platform that enables asynchronous conversations and a unified agent and supervisor experience across all Genesys Cloud messaging channels. Messenger is the predefined message window that customers use to communicate with bots and agents.
```

Comment:
Are you actually using Web messaging and message directly in the flow.

Reply:
No but I use it in testing